src/main/java/com/daedafusion/security/bindings/AuthorizationTokenFilter.java

```diff
@@ -73,7 +71,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             // Strip "Bearer " if present
             authorizationToken = authorizationToken.replaceAll("(?i)"+ Pattern.quote("Bearer "), "");
 
-            Token token = tokenExchange.getToken(authorizationToken);
+            ContextToken token = new ContextToken(authorizationToken);
+            token.addContext("client-ip", httpServletRequest.getHeader("X-Forwarded-For"));
```

Comment:
This probably shouldn't be here since this filter is meant to be generic. 

Alternatively, I could pass all the headers from the request into the context...